src/cljs/org/broadinstitute/firecloud_ui/page/workspaces.cljs

```diff
@@ -317,7 +326,10 @@
        "/workspaces"
        {:on-done (fn [{:keys [success? xhr]}]
                    (if success?
-                     (let [workspaces (utils/parse-json-string (.-responseText xhr))]
+                     (let [raw-workspaces (utils/parse-json-string (.-responseText xhr))
+                           ;; The following is just to make the icons work with the
+                           ;; current live data, which uses this missing field
+                           workspaces (map #(assoc % "status" "Complete") raw-workspaces)]
```

Comment:
I think this modification should be conditional on `utils/use-live-data`. Otherwise, I still can't see the nice icons unless I also modify the code here. This is hacky, and I'm not even sure if Rawls will ever return status like this, but let's at least make it easy to expose them for now.